src/annotations/modifiers/AnnotationItemModifier.cpp

```diff
@@ -149,8 +149,8 @@ void AnnotationItemModifier::handleSelection()
 		clear();
 	} else {
 		mItemResizer->attachTo(selectedItems);
-		emit itemsSelected(selectedItems);
 	}
+	emit itemsSelected(selectedItems);
```

Comment:
When deselecting item, will call `AnnotationArea::itemsSelected` with no items selected, which in turn will activate the select tool.

Reply:
Shouldn't we introduce here a new signal, like deselectItems or something like that, in the end it might just call the activateSelectTool but IMHO it might be more obvious whats going on.

Reply:
Thank you. Will create another signal for this case.